### PR TITLE
Create wishlist html table and complete js setup

### LINKED ISSUE #3: Convert wishlist into HTML using AirTable API and complete JS setup
- [x] Get sourcemaps working
- [x] Remove current AirTable embed
- [x] Pull information in using AirTable API and React instead

### PR DESCRIPTION
**Pull request by ghost**
Now displaying wishlist data from AirTable in a basic Material-UI React component.

Also, developers can now store their own AirTable API keys in /.airtable/apiKey.js and it will be used as the AIRTABLE_API_KEY environment variable.

### CODE REVIEW

**Comment by ghost** [comment]
Updated so that if apiKey is not found in either environment variables or in local file then the build fails

**Comment by kimadactyl** [comment]
This looks great, only thing I would add is instructions in the README for how to add your AirTable key to your local account (and GitHub).

**Comment by kimadactyl** [comment]
Oh and let's revert back to vanilla HTML5 to make it easier when I get around to doing a theme!

**Comment by kimadactyl** [comment]
I've also just changed the field spec slightly now this is happening to make it more consistent. Should pull Name, Email, Phone in the new setup.

**Comment by ghost** [comment]
Have reverted to vanilla HTML and changed columns to Name, Email and Phone

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "bundle.js",
   "scripts": {
     "build": "webpack",
-    "start": "yarn build && jekyll serve",
+    "start": "yarn watch && jekyll serve --livereload",
```
Comment (ghost) [inline package.json]: i think this will start jekyll serve twice, one with and one without live reload.

**Pull request merged**